ide/coqide/wg_ProofView.ml

```diff
@@ -68,11 +68,11 @@ let mode_tactic sel_cb (proof : #GText.view_skel) goals ~unfoc_goals hints = mat
       let head_str = Printf.sprintf
         "%d subgoal%s\n" goals_cnt (if 1 < goals_cnt then "s" else "")
       in
-      let goal_str ?(shownum=false) index total =
-        if shownum then Printf.sprintf
-                          "______________________________________(%d/%d)\n" index total
-        else Printf.sprintf
-               "______________________________________\n"
+      let goal_str ?(shownum=false) index total id =
+        let annot =
+          if id = "" then if shownum then Printf.sprintf "(%d/%d)" index total else ""
```

Comment:
Use String.is_empty.

Reply:
Would you be interested in using the suggestion feature?

Reply:
Overall it takes more time for everybody since you'll have to squash the commit anyways. I can do it if you insist.

Reply:
```suggestion
          if CString.is_empty id then if shownum then Printf.sprintf "(%d/%d)" index total else ""
```

Reply:
My experience is that it often reduces the interaction needed to check that we did what the other expected. It also allow me to validate the changes and to squash in independent steps. But if it takes more time for you, I don't insist.